terraform/azure.tf

```
@@ -620,3 +620,29 @@ resource "azurerm_sql_database" "sql-database" {
   depends_on          = ["azurerm_sql_server.sql-server"]
   tags {}
 }
+
+resource "azurerm_kubernetes_cluster" "test" {
+  name                = "inspecaks"
```

```suggestion
  name                = "inspecakstest"
```

The `agent_pool_profile` name should be unique from the `azurerm_kubernetes_cluster` name. When they are the same tests were failing.